src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/Commands/MainListPage.cs

```diff
@@ -149,13 +150,13 @@ private bool ActuallyLoading()
     // _always_ show up first.
     private int ScoreTopLevelItem(string query, IListItem topLevelOrAppItem)
     {
-        if (string.IsNullOrWhiteSpace(query))
+        if (string.IsNullOrEmpty(query))
```

Comment:
After:

For **only an empty filter**, we give everything a weight of 1. Just a whitespace filter will now fall through, and be used to weight & filter items...